.github/workflows/cve-check.yml

```diff
@@ -0,0 +1,32 @@
+name: CVE Check
+
+on:
+  push:
+    branches:
+      - master
+      - release/2.x.x
+  pull_request:
+
+jobs:
+  trivy-vulnerability-check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3
+
+      - name: Setup Gradle
+        uses: gradle/gradle-build-action@v2.5.0
```

Comment:
Our other workflows use `setup-java`. What's the difference / benefit to use this action instead? Should we streamline the other workflows?

See https://github.com/SDA-SE/sda-dropwizard-commons/blob/master/.github/workflows/java-ci.yml#L29, for example.